<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="edit_DisplayLog">
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rdo_2M">
        <property name="text">
         <string>2Mbps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdo_1M">
        <property name="text">
         <string>1Mbps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdo_500K">
        <property name="text">
         <string>500Kbps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdo_125K">
        <property name="text">
         <string>125Kbps</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tag:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_Tag"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_GetLocation">
        <property name="text">
         <string>Get Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Export">
        <property name="text">
         <string>Export to Clipboard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
